deploy/005_fee_manager.ts

```diff
@@ -34,17 +34,15 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
       throw new Error(`Unsupported chain '${hre.network.name}`);
   }
 
-  const hub = await hre.deployments.get('DCAHub');
-
   await deployThroughDeterministicFactory({
     deployer,
     name: 'DCAFeeManager',
     salt: 'MF-DCAV2-DCAFeeManager-V2',
```

Comment:
We had already updated this, right?

Reply:
Maybe? Haha. We will know for sure when we try to deploy